saleor/settings.py

```diff
@@ -671,7 +671,13 @@ def get_host():
 
 EXTENSIONS_MANAGER = "saleor.extensions.manager.ExtensionsManager"
 
+PAYMENT_PLUGINS = [
```

Comment:
This should be taken from env

Reply:
As I spoke with @maarcingebala , we're not convinced it is the best idea to inject python module paths through PATH, since now we still need to set thing up in saleor settings, will leave it manually created here.
After we implement automatic plugin discovery we might consider configuring additional plugin paths through env.